src/sagemaker/pytorch/model.py

```diff
@@ -136,7 +136,7 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
                 For example, 'ml.p2.xlarge'.
             accelerator_type (str): The Elastic Inference accelerator type to
                 deploy to the instance for loading and making inferences to the
-                model. Currently unsupported with PyTorch.
+                model. Currently supported with PyTorch 1.3.1 Python 3.
```

Comment:
nit: I would just remove "Currently unsupported with PyTorch" entirely and leave the version info for the READMEs.

Reply:
removed.